pkg/lib/bundle/validate_test.go

```diff
@@ -36,17 +29,10 @@ func TestPullBundle_Error(t *testing.T) {
 
 	expectedErr := fmt.Errorf("Unable to unpack image")
 
-	logger := logrus.NewEntry(logrus.New())
-
 	mockImgReader := containertoolsfakes.FakeImageReader{}
 	mockImgReader.GetImageDataReturns(expectedErr)
 
-	validator := imageValidator{
-		imageReader: &mockImgReader,
-		logger:      logger,
-	}
-
-	err := validator.PullBundleImage(tag, dir)
+	err := mockImgReader.GetImageData(tag, dir)
```

Comment:
This is also no longer testing anything.

Reply:
This never did anything but to pass through an error. As mentioned in the last comment, this will stay even though it is testing passing error.